### ISSUE TITLE
Say so when a trail is exhausted

### ISSUE THREAD

**Issue by henrik** [exercism/cli#53]
`exercism fetch` now tells me:

```
$ exercism fetch
clojure - bob
…
python - word-count
ruby - matrix
…
ruby - largest-series-product
scala - word-count
Exercises written to /Users/henrik/Dropbox/Code/exercism
```

Before, the next Ruby exercise up has appeared as the bottom of the list of Ruby exercises, but now it lists largest-series-product which I've already done. I suspect this is because I've exhausted the trail, or the trail order has changed or something.

Maybe the tool could say something like "No more exercises in the Ruby trail! Try another trail, help review submissions or contribute an exercise of your own."

**Comment by kytrinyx** [comment]
The status on this ticket is that we're waiting for some work in x-api and exercism before we can reimplement the `fetch` command to call a different endpoint that has the data we need for this.

**Comment by Insti** [comment]
I believe this issue is ready to be revived.

Ref: https://github.com/exercism/exercism.io/issues/2156#issuecomment-255624815

**Comment by kytrinyx** [comment]
We have something in the exercism.io profile page, but not the x-api, I think. The CLI doesn't have anything to consume yet.

**Comment by Insti** [comment]
I suspect probot is a little over enthusiastic for this repository. Just because things are not fixed quickly doesn't mean they should not be fixed.
